zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -9,7 +9,8 @@ acsrf.api.action.removeOptionToken = Removes the anti-CSRF token with the given
 acsrf.api.action.removeOptionToken.param.String = 
 acsrf.api.desc =
 acsrf.api.other.genForm = Generate a form for testing lack of anti-CSRF tokens - typically invoked via ZAP
-acsrf.api.other.genForm.param.hrefId = 
+acsrf.api.other.genForm.param.hrefId =
+acsrf.api.other.genForm.param.actionUrl =
```

Comment:
You could provide an actual description here. In fact you could embrace "clean code" and add one for hrefId as well.

Reply:
fixed